### ISSUE TITLE
Negative filters

### ISSUE THREAD

**Issue by szepeviktor** [bonny/WordPress-Simple-History#86]
Could we have negative filter checkboxes?

NOT containing words

NOT log level debug etc.

**Comment by bonny** [comment]
yeah maybe. I never needed it myself. Can you give some examples of when you felt the need for negative filters?

**Comment by szepeviktor** [comment]
There is a site with one user. I'd like to all other activities.

**Comment by szepeviktor** [comment]
@bonny Please consider implementing the negative filter.

A checkbox would be nice.
Thank you!

**Comment by szepeviktor** [comment]
@bonny Hello!

Have you came across this issue?

**Comment by bonny** [comment]
Yes I also think this would be a good feature. I will look at it if I find time.

**Comment by bonny** [comment]
Related #412 #236

**Comment by szepeviktor** [comment]
@bonny A decade, Bonny, a decade.

**Comment by bonny** [comment]
@szepeviktor Slow progress is better than no progress! :)
